code/math/vecmat.cpp

```diff
@@ -1397,7 +1397,7 @@ void compute_point_on_plane(vec3d *q, const plane *planep, const vec3d *p)
 
 	vm_vec_scale_add(q, p, &normal, -k);
 
-	tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
+	// tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
```

Comment:
The last change in that line was in 2008 and I can't blame any further because the the GitHub interface is horrible.